src/app/clients/ActionBar.cy.tsx

```diff
@@ -3,122 +3,243 @@ import React from "react";
             parcelId: "123456789",
             requiresFollowUpPhoneCall: false,
+            flaggedForAttention: false,
+            voucherNumber: "123456789",
         },
         {
             primaryKey: "primaryKey2",
             addressPostcode: "AB1 aaaa2CD",
             collectionCentre: "Centraaaae 1",
+            congestionChargeApplies: true,
             collectionDatetime: new Date().toISOString(),
-            congestionChargeApplies: false,
             familyCategory: "Familaaaay 1",
-            flaggedForAttention: false,
             fullName: "John Smaaaaith",
-            lastStatusUpdate: new Date().toISOString(),
             lastStatus: "Deliveaaaared",
             packingDatetime: new Date().toISOString(),
             packingTimeLabel: "10aaaa:00",
             parcelId: "123456aaaa789",
             requiresFollowUpPhoneCall: false,
+            flaggedForAttention: false,
+            voucherNumber: "123456aaaa789",
         },
     ];
 
-    const selectedIndices = [0, 1];
+    const mockClientsData: Schema["clients"] = {
+        address_1: "123",
+        address_2: "Gem Lane",
+        address_county: "Gemmashire",
+        address_postcode: "GE33AA",
+        address_town: "Gemm",
+        baby_food: null,
+        delivery_instructions: "Get some gem",
+        dietary_requirements: ["I eat gems"],
+        extra_information: "Gems",
+        family_id: "789456123",
+        feminine_products: ["Gems"],
+        flagged_for_attention: false,
+        full_name: "John Smith",
+        other_items: [],
+        pet_food: [],
+        phone_number: "0123456789",
+        primary_key: "primeKey",
+        signposting_call_required: false,
+    };
+
+    const mockFamiliesData: Schema["families"] = {
+        age: 1,
+        family_id: "789456123",
+        gender: "female",
+        primary_key: "7894561235",
+    };
 
-    const MockActionBar: React.FC<{}> = () => {
+    const mockListsData: Schema["lists"] = {
+        "1_notes": null,
+        "1_quantity": "1",
+        "10_notes": null,
+        "10_quantity": "1",
+        "2_notes": null,
+        "2_quantity": "1",
+        "3_notes": null,
+        "3_quantity": "1",
+        "4_notes": null,
+        "4_quantity": "1",
+        "5_notes": null,
+        "5_quantity": "1",
+        "6_notes": null,
+        "6_quantity": "1",
+        "7_notes": null,
+        "7_quantity": "1",
+        "8_notes": null,
+        "8_quantity": "1",
+        "9_notes": null,
+        "9_quantity": "1",
+        item_name: "Eggs",
+        primary_key: "012345689",
+        row_order: 5,
+    };
+
+    interface Props {
+        selected: number[];
+    }
+
+    const MockActionBar: React.FC<Props> = ({ selected }) => {
         return (
             <Localization>
                 <StyleManager>
-                    <ActionBar data={mockData} selected={selectedIndices} />
+                    <ActionBar {...{ data: mockData, selected }} />
```

Comment:
why do we need to change this?

Reply:
I wasn't the one to change this so I do not know why. I changed it back now cause I think the older version is more readable.